docs/WINDOWS.md

```
@@ -0,0 +1,124 @@
+
+On a default Windows install, `bash` on `PATH` resolves to `C:\WINDOWS\System32\bash.exe` — that is the **WSL launcher**, not Git Bash. It exists even if you have never knowingly used WSL.
+
+Check which one you have:
+
+```powershell
+Get-Command bash | Format-List Source
+```
+
+If that prints `C:\WINDOWS\System32\bash.exe`, anything that shells out through bare `bash` runs inside a Linux distribution instead of Windows. WSL has its own `PATH`, so the Windows `node` is invisible there. `batch/batch-runner.sh` then dies partway through with:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)